modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenResponse.java

```diff
@@ -79,30 +113,152 @@ public boolean equals(Object o) {
-                hasHeaders, isString, isNumeric, isInteger, isLong, isNumber, isFloat, isDouble, isByteArray,
-                isBoolean, isDate, isDateTime, isUuid, isEmail, isModel, isFreeFormObject, isDefault, simpleType,
-                primitiveType, isMapContainer, isListContainer, isBinary, isFile, schema, jsonSchema, vendorExtensions);
+    public String getPattern() {
+        return pattern;
+    }
+
+    @Override
+    public void setPattern(String pattern) {
+        this.pattern = pattern;
+    }
+
+    @Override
+    public String getMaximum() {
+        return maximum;
+    }
+
+    @Override
+    public void setMaximum(String maximum) {
+        this.maximum = maximum;
+    }
+
+    @Override
+    public String getMinimum() {
+        return minimum;
+    }
+
+    @Override
+    public void setMinimum(String minimum) {
+        this.minimum = minimum;
+    }
+
+    @Override
+    public boolean getExclusiveMaximum() {
+        return exclusiveMaximum;
+    }
+
+    @Override
+    public void setExclusiveMaximum(boolean exclusiveMaximum) {
+        this.exclusiveMaximum = exclusiveMaximum;
+    }
+
+    @Override
+    public boolean getExclusiveMinimum() {
+        return exclusiveMinimum;
+    }
+
+    @Override
+    public void setExclusiveMinimum(boolean exclusiveMinimum) {
+        this.exclusiveMinimum = exclusiveMinimum;
+    }
+
+    @Override
+    public Integer getMinLength() {
+        return minLength;
+    }
+
+    @Override
+    public void setMinLength(Integer minLength) {
+        this.minLength = minLength;
+    }
+
+    @Override
+    public Integer getMaxLength() {
+        return maxLength;
+    }
+
+    @Override
+    public void setMaxLength(Integer maxLength) {
+        this.maxLength = maxLength;
+    }
+
+    @Override
+    public Integer getMinItems() {
+        return minItems;
+    }
+
+    @Override
+    public void setMinItems(Integer minItems) {
+        this.minItems = minItems;
+    }
+
+    @Override
+    public Integer getMaxItems() {
+        return maxItems;
+    }
+
+    @Override
+    public void setMaxItems(Integer maxItems) {
+        this.maxItems = maxItems;
+    }
+
+    @Override
+    public boolean getUniqueItems() {
+        return uniqueItems;
+    }
+
+    @Override
+    public void setUniqueItems(boolean uniqueItems) {
+        this.uniqueItems = uniqueItems;
+    }
+
+    @Override
+    public Integer getMinProperties() {
+        return minProperties;
+    }
+
+    @Override
+    public void setMinProperties(Integer minProperties) {
+        this.minProperties = minProperties;
+    }
+
+    @Override
+    public Integer getMaxProperties() {
+        return maxProperties;
+    }
+
+    @Override
+    public void setMaxProperties(Integer maxProperties) {
+        this.maxProperties = maxProperties;
     }
 
     @Override
     public String toString() {
-        final StringBuffer sb = new StringBuffer("CodegenResponse{");
```

Comment:
FYI. This is auto-generated by Intelllji IDEA

Reply:
Are you commenting about the removed line or my modified line? My line was my IntelliJ default auto-generated output.